### DISCUSSION TITLE
security onion salt services  wait_start

### DISCUSSION THREAD

**Discussion by farshidmosaffa** [Security-Onion-Solutions/securityonion#8230] [Unsupported Versions]
hi. 
I used SO ISO and the node is master&search and using last version.
my server was working until I restart it. all the time it is wait_start or starting.  firewall is inactive on server. by so-***-restart --force some come up and status become "OK". but some fail.
EX. curator service was in wait_start state, I run so-curator-restart.  but some of the even after fore restart get failed.
would u please help?
THX

by the way, is there any solution to restart all service together?
thanks again
![Screen Shot 2022-07-04 at 9 03 35 PM](https://user-images.githubusercontent.com/108687213/177193197-6ebbb1ed-0940-41a0-a9e1-f2edfa129bf7.png)

**Comment by popokoko** [comment]
You can refer to (https://github.com/Security-Onion-Solutions/securityonion/discussions/8013)
